src/slide-up-down.js

```diff
@@ -30,52 +28,33 @@ export default {
     )
   },
 
-  mounted () {
-    this.layout()
-
-    window.addEventListener('resize', this.layout)
-
-    setTimeout(() => {
-      this.isMounted = true
```

Comment:
`isMounted` is necessary for the case where `active === true` while mounting, because otherwise the elements will animate from closed to open position.

Reply:
In my first post, I threw off the gif and there just is a bug with the animation. isMounted not working.

Reply:
Feel free to keep your own fork!